tests/installation/start_install.pm

```diff
@@ -48,6 +48,7 @@ sub run {
                 while (!check_screen('all-conflicts-resolved-packages', 4)) {
                     assert_and_click 'package-conflict-choice';
                     send_key $cmd{ok};
+                    wait_still_screen 10;
```

Comment:
Why do we need to wait a full 10 seconds here? How about changing the whole block with a multi-tag assert_screen and match_has_tag in a loop? You are getting good at the needling witchcraft so I trust you to do that :-)